Sources/CvPlot.cpp

```diff
@@ -11839,7 +11771,7 @@ void CvPlot::write(FDataStreamBase* pStream)
 	}
 	else
 	{
-		WRAPPER_WRITE_DECORATED(wrapper, "CvPlot", (char)MAX_PLAYERS, "cFoundValuesPresent");
+		WRAPPER_WRITE_DECORATED(wrapper, "CvPlot", (char)1, "cFoundValuesPresent");
```

Comment:
This will bug out on the other end without further changes:
```cpp
	WRAPPER_READ(wrapper, "CvPlot", &cFoundValuesPresent);
	if ( cFoundValuesPresent != (char)-1 )
	{
		unsigned int* m_aiFoundValue = new unsigned int[cFoundValuesPresent];
		WRAPPER_READ_ARRAY(wrapper, "CvPlot", cFoundValuesPresent, m_aiFoundValue);
	}
	else
```

Reply:
why?

Reply:
Because you changed cFoundValuesPresent to equal 1 instead of MAX_PLAYERS.
m_aiFoundValue = new unsigned int[1] is not the same as m_aiFoundValue = new unsigned int[MAX_PLAYERS]

Reply:
wasn't paying that much attention.